lib/node_modules/@stdlib/string/base/format-tokenize/test/test.js

```diff
@@ -271,3 +271,36 @@ tape( 'the function tokenizes a string (extracting precision)', function test( t
 
 	t.end();
 });
+
+tape( 'the function tokenizes a string (extracting escaped percent sign)', function test( t ) {
```

Comment:
Another test I suggest adding is where the escaped percent sign comes before another specifier. E.g.,

```
'%%%d'
```

where the result should be, e.g., `%10`.

Reply:
Similarly, to be pedantic, maybe also double escaped percent signs: `%%%%`.